tensorflow/contrib/all_reduce/python/all_reduce.py

```diff
@@ -317,7 +317,7 @@ def _build_ring_gather(input_tensors, devices, num_subchunks,
   num_devices = len(input_tensors)
   if num_devices == 0:
     return []
-  if num_devices == 1:
+  elif num_devices == 1:
```

Comment:
Why change this? The existing version seems equivalent, no?

Reply:
Two times `if-statement` was being shown using the same variable. That's why thought to put it in with `elif` statement. Should I revert it back?

Reply:
Yes please. Not that there is anything wrong with the `elif`, but then again, not there was nothing wrong with the `if` either :) So in the interest of avoiding superfluous changes, let's revert this.